test/collection.js

```diff
@@ -142,4 +142,12 @@ describe('Collection interface', function() {
       assert.ok(documentRef.id.length, 20);
     });
   });
+
+  it('has equals() method', function() {
```

Comment:
equals() => isEqual() here and elsewhere?

Reply:
Done.